staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go

```diff
@@ -62,6 +62,9 @@ type RequestInfo struct {
 	Name string
 	// Parts are the path parts for the request, always starting with /{resource}/{name}
 	Parts []string
+
+	// UserAgent is a value of 'User-Agent' header, if set.
+	UserAgent string
```

Comment:
I would like to hear @liggitt ` opinion on extending RequestInfo type.

Reply:
in general, I'm not a huge fan of using unverified input data like user-agent headers

Reply:
Dropped